sample/sample-with-di/app/src/main/AndroidManifest.xml

```diff
@@ -9,12 +9,15 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name="com.dropbox.kaiken.sample.advancedsample.app.MainActivity">
+        <activity android:name="com.dropbox.kaiken.sample.advancedsample.app.LoggedOutActivity">
```

Comment:
nit: When I read `LoggedOutActivity` my initial thought is it is used after the user has been logged out.  Maybe just `SplashActivity`?

Reply:
+1